librz/bin/dwarf/loclists.c

```diff
@@ -415,7 +379,7 @@ RZ_API void rz_bin_dwarf_location_free(RZ_BORROW RZ_NONNULL RzBinDwarfLocation *
  * \param self RzBinDwarfLocation instance
  * \return RzBinDwarfLocation instance on success, NULL otherwise
  */
-RZ_API RZ_OWN RzBinDwarfLocation *rz_bin_dwarf_location_clone(RZ_BORROW RZ_NONNULL RzBinDwarfLocation *self) {
+RZ_API RZ_OWN RzBinDwarfLocation *rz_bin_dwarf_location_clone(RZ_BORROW RZ_NONNULL Location *self) {
```

Comment:
This is the API function, I wonder if it's better to keep the `Location` type name the old way, since it's not a generic concept, if I am not wrong.